src/Game.cs

```diff
@@ -183,6 +183,30 @@ public GameWindow Window
 
 		internal bool RunApplication;
 
+		/* Setup Text Input Control Character Arrays
```

Comment:
Hard to say what's the best spot for this, but my gut says FNAPlatform. They can definitely be `static readonly` no matter where it ends up though.